src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java

```diff
@@ -22,6 +22,7 @@ public final class AccountAttributes extends EntityAttributes<Account> {
     private String email;
     private Map<String, Instant> readNotifications;
     private Instant createdAt;
+    private boolean isMigrated;
 
     private AccountAttributes(String googleId) {
         this.googleId = googleId;
```

Comment:
Should we add `this.isMigrated = false` here?

Reply:
I think the default value for `isMigrated` is false, so we might not need that